ert_shared/status/tracker/evaluator.py

```diff
@@ -36,30 +37,18 @@ class EvaluatorTracker:
     def __init__(
         self,
         model: BaseRunModel,
-        host,
-        port,
-        token=None,
-        cert=None,
+        ee_config: EvaluatorServerConfig,
```

Comment:
`EvaluatorConnectionInfo` ?

Reply:
Yes! Will get to refactor everything now. But this is exactly it. : ) 👍